app/services/statements/search.rb

```diff
@@ -50,6 +56,13 @@ def where_statement_date(statement_date)
       @scope = scope.with_statement_date(year:, month:)
     end
 
+    def where_deadline_date(deadline_date)
+      return if ignore?(filter: deadline_date)
+      return if deadline_date.blank?
```

Comment:
I think this check is redundant as the `FilterIngorable` concern will treat blank strings as ignored

Reply:
Hi @ethax-ross For some reason `FilterIgnorable` wasn't ignoring `nil` values.
I've fixed that now. I hope it's all good. tks

Reply:
Ah I'm forgetting `blank` checks for `nil` as well -- I think that's why we have `:ignore` so that you _can_ pass in `nil` and have it filter. I wouldn't change `FilterIgnorable` as its used in a bunch of places in the API as well - sorry, lets maybe just leave it as you had it 😅